CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Most standard mathematical functions are now supported: `abs`, `floor`,
   `ceil`, `pi`, `exp`, `ln`, `log10`, `log`, `sqrt`, `degrees`, `radians`,
   `cos`, `acos`, `sin`, `asin`, `tan`, `atan` and `pow` (@PrettyWood, #3909)
+- Those mathematical functions are in the `math` module (@PrettyWood, #3916)
```

Comment:
If you have the patience — let's integrate this with the item above, since they can be handled as one change. Fine to list more than one PR under one change.

Reply:
Done in ccc424d